apps/storybook/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "storybook",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently \"yarn:watch:*\"",
```

Comment:
this needs yarn installed? will we be able to reuse pnpm if that's what turbo is set up with?

Reply:
true. will update to pnpm.